src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -235,10 +249,13 @@ protected void runTool( final JavaSparkContext ctx ) {
             final Locations locations,
             final PipelineOptions pipelineOptions )
     {
+        Set<Integer> crossContigsToIgnoreSet;
+        if ( crossContigsToIgnoreFile != null ) crossContigsToIgnoreSet = Collections.emptySet();
```

Comment:
Oh, wow.  So this has never actually been tested.  The 5% variation I saw was just noise.
Thanks, Steve.  I'll make this fix and test before merging.  Thanks for catching this.